docs/src/appendix/scarb-toml.md

```diff
@@ -158,14 +185,17 @@ The `starknet-contract` target allows to build the package as a Starknet Contrac
 ```
 
 #### `sierra`
+
 See more about [Sierra contract class generation](https://docs.swmansion.com/scarb/docs/extensions/starknet/contract-target.html#sierra-contract-class-generation) in Scarb documentation.
+
 ```toml
 [[target.starknet-contract]]
 sierra = true
 ```
 
 #### `casm`
 
+Enabling [`casm = true`](https://github.com/foundry-rs/starknet-foundry/releases/tag/v0.23.0) in Scarb.toml causes unnecessary overhead and should be disabled unless required by other tools. Tools like `snforge` and `sncast` recompile Sierra to CASM separately, resulting in redundant processing. This duplicates CASM generation, significantly impacting performance, especially for large Sierra programs.
```

Comment:
```suggestion
Enabling [`casm = true`](https://github.com/foundry-rs/starknet-foundry/releases/tag/v0.23.0) in Scarb.toml causes unnecessary overhead and should be disabled unless required by other tools. Starknet Foundry compiles Sierra to CASM separately, resulting in redundant processing. This duplicates CASM generation, significantly impacting performance, especially for large Sierra programs.
```

Reply:
Hello @cptartur  and @ksew1,

Thank you both for your feedback. @cptartur, I have committed your suggestion. I noticed that your final suggestion still includes a link to v0.23.0. Could you confirm if it’s fine to keep this link, or would you prefer I remove it?